tools/run_tests/sanity/check_port_platform.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+
+# Copyright 2017 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+
+os.chdir(os.path.join(os.path.dirname(sys.argv[0]), '../../..'))
+
+
+def check_port_platform_inclusion(directory_root):
+    bad_files = []
+    for root, dirs, files in os.walk(directory_root):
+        for filename in files:
+            path = os.path.join(root, filename)
+            if os.path.splitext(path)[1] not in ['.c', '.cc', '.h']: continue
+            if 'port_platform.h' in filename: continue
+            if 'load_balancer.pb.h' in filename: continue
+            if 'load_balancer.pb.c' in filename: continue
+            with open(path) as f:
+                for l in f.readlines():
+                    if '#include' in l:
+                        if 'port_platform' not in l:
```

Comment:
Let's be more specific here.  We should make sure the include is exactly correct, so that if anyone ever adds a file in another directory with the same name, there's no confusion.

Also, let's make sure the line after this one is a blank line.

Reply:
addressed both comments here; the recent update makes the checks more specific and requires that the full line matches as expected